client/state/experiments/reducer.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External Dependencies
+ */
+import { findIndex, tail } from 'lodash';
+import { Action } from 'redux';
+
+/**
+ * Internal Dependencies
+ */
+import { EXPERIMENT_FETCH, EXPERIMENT_ASSIGN, CURRENT_USER_RECEIVE } from 'state/action-types';
+import { ExperimentState, ExperimentAssign } from 'state/experiments/types';
+
+/**
+ * Attempt to get the anon id for the user, if set
+ */
+function getAnonIdFromCookie(): string | null {
+	if ( document && document.cookie != null ) {
+		const cookieString = document.cookie;
+		const rawCookies = cookieString.split( ';' );
+		const anonCookieIndex = findIndex( rawCookies, c =>
+			c
+				.trim()
+				.toLowerCase()
+				.startsWith( 'tk_ai=' )
+		);
+		if ( anonCookieIndex === -1 ) return null;
+
+		const anonIdCookie = rawCookies[ anonCookieIndex ];
+		const anonIdTuple = anonIdCookie.split( '=' );
+
+		if ( anonIdTuple.length === 2 ) return anonIdTuple[ 1 ] === '' ? null : anonIdTuple[ 1 ];
+		if ( anonIdTuple.length > 2 ) return tail( anonIdTuple ).join( '=' );
+	}
+	return null;
+}
+
+const appStartedAt = Date.now();
+
+const resetState: ( anonId: string | null ) => ExperimentState = anonId => ( {
+	anonId,
+	isLoading: true,
+	nextRefresh: appStartedAt,
+	tests: null,
```

Comment:
`tests` -> `experiments` or `variations`? :slightly_smiling_face: (here and everywhere else)

`variations` may be better given the API response.

Reply:
`variations` is better. I was just fumbling around so it wouldn't be `experiments.experiments`